### PR TITLE
repeater base implementation

### PR DESCRIPTION
**Pull request by Mboulianne**
## `@ulaval/modul-components`
# PR Checklist

- [x] Provide a small description of the changes introduced by this PR
Add a basic implementation for repeater component
- [x] Include links to issues
https://jira.dti.ulaval.ca/browse/MODUL-929
- [ ] Openshift deployment requested

- [ ] Include this section in the release notes

- [ ] Other info

### CODE REVIEW

**Comment by Mboulianne** [comment]
@Atiomi  ça va prendre une intégration de base :)

File: src/components/repeater/repeater.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    }
+
+    onAddBtnClick(): void {
+        this.$emit('add');
+    }
+
+    onDeleteBtnClick(index: number): void {
+        this.$emit('delete', index);
+    }
+
```
Comment (fredduguay) [inline src/components/repeater/repeater.ts]: Utiliser `@Emit` pour  add et delete

Comment (Mboulianne) [reply src/components/repeater/repeater.ts]: DOne

**Comment by Mboulianne** [comment]
@Atiomi qu'est-ce que tu veux dire?

**Pull request merged**